theano/gpuarray/sort.py

```diff
@@ -100,9 +113,11 @@ def gpu_kernels(self, node, nodename):
         param_types = [ga.SIZE] * (ndim - 1)  # dims
         for _ in range(self.return_values + self.return_indices):
             param_types.append(ga.GpuArray)  # dst*
+            param_types.extend([ga.SIZE] * ndim)  # offset
```

Comment:
This line should be `param_types.append(ga.SIZE)`

Reply:
Thanks, this fix one error I got yesterday. I pushed this fix. Now I have another error CUDA_ERROR_LAUNCH_OUT_OF_RESOURCES. I'll check later today, I'm in meeting most of the day.

Reply:
`CUDA_ERROR_LAUNCH_OUT_OF_RESOURCES` triggers when the kernel use too much registers in GPU. Computing array+offset on CPU side, therefore reducing kernel argument count might help.

Reply:
We can't use too many registers since we limit register usage for the specific card that we are using.

However, it might be the case that we try to launch too many threads at once which might cause this.